src/common/low_precision_transformations/tests/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME ov_lp_transformations_tests)
+
+if(CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
```

Comment:
Do we really have all these warnings?

Reply:
> Do we really have all these warnings?

I just copied this from ieFuncTests to not spend a lot of time to mute each warning.
Anyway next step is fix all of them